doc/languages-frameworks/rust.section.md

```diff
@@ -55,20 +55,9 @@ used for [SRI](https://www.w3.org/TR/SRI/) hashes. For example:
 ```
 
 Both types of hashes are permitted when contributing to nixpkgs. The
-Cargo hash is obtained by inserting a fake checksum into the
-expression and building the package once. The correct checksum can
-then be taken from the failed build. A fake hash can be used for
-`cargoSha256` as follows:
-
-```nix
-  cargoSha256 = lib.fakeSha256;
-```
-
-For `cargoHash` you can use:
-
-```nix
-  cargoHash = lib.fakeHash;
-```
+Cargo hash is obtained by building the package once without
+`cargoSha256` or `cargoHash` set. The correct checksum can then be
+taken from the failed build.
```

Comment:
Why did you remove this? PR [#191378](https://github.com/NixOS/nixpkgs/pull/191378/files#diff-99f4eb67c3def9d78bbb9ec86a7dced190f46ee7151c77dcbb15a6a08727a330R483) seems relevant.